.chloggen/remove_confmap.expandEnabled_stable.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Do we need a changelog for this? When the gate is `Stable` setting it at all, either true or false, causes the collector to not start, so this PR isn't a breaking change.

Reply:
I can remove, I wasn't sure.

Reply:
done!